data/SL-Hammocks/pop_salp_a0001_zl001_to_5_wo_sub.csv

```diff
@@ -0,0 +1,326 @@
+,zl,m_h,m_acc,e_h,p_h,con,m_g,e_g,p_g,tb,vel_disp
```

Comment:
can you add a short readme of what this csv file is and what the columns mean? Otherwise we might risk accidentally removing it or changing it in undesired ways